### PR TITLE
Setup project using vue-cli

### PR DESCRIPTION
**Pull request by rejas**
The old boilerplate used for setting the project up is outdated as are the dependencies.

Therefor I re-setup the project using the official vue-cli.

Fixed the linting errors while on it.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,18 +0,0 @@
-<!doctype html>
```
Comment (dutscher) [inline docs/index.html]: no delete, use this for ghpages

File: package.json [hunk 2]
```diff
@@ -5,57 +5,38 @@
   "author": "Willy Woitas <willy.woitas@neofonie.de>",
   "private": true,
   "scripts": {
-    "npmInstall": "npm install",
```
Comment (dutscher) [inline package.json]: dont delete my scripts!! kollege

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (dutscher) [inline public/index.html]: dont commit dist shit

File: src/components/ClrPikr.vue [hunk 4]
```diff
@@ -1,67 +1,70 @@
...
+      v-on:change="onChange($event, badgeKey, type)"
+    ></color-picker>
   </span>
 </template>
 
 <script>
-  // https://github.com/v-comp/v-color/blob/master/demo.js
-  import ColorPicker from '../../node_modules/v-color/dist/index.esm';
+// https://github.com/v-comp/v-color/blob/master/demo.js
+import ColorPicker from "../../node_modules/v-color/dist/index.esm";
```
Comment (dutscher) [inline src/components/ClrPikr.vue]: single quotes bitte

File: docs/index.html (hunk 1 above)
Comment (rejas) [reply docs/index.html]: docs are back in again

File: package.json (hunk 2 above)
Comment (rejas) [reply package.json]: willys scripts are back again

File: src/components/ClrPikr.vue (hunk 4 above)
Comment (rejas) [reply src/components/ClrPikr.vue]: done!

File: public/index.html (hunk 3 above)
Comment (rejas) [reply public/index.html]: thats not dist shit, thats the template used for building the dist shit

**Pull request merged**